components/EditEventForm.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { defineMessages, injectIntl } from 'react-intl';
-
```

Comment:
We already have a `slugify` package in our `package.json`, let's use this one?

Reply:
We use `limax` (which has better support for special characters, https://github.com/opencollective/opencollective-api/pull/1480#issuecomment-429791678) on the API, wouldn't be a bad idea to make the switch for the Frontend.

But I agree that this should be part of a separate PR dedicated to that, we should use `slugify` for now.